src/test/java/org/jenkinsci/test/acceptance/recorder/TestRecorderRuleTest.java

```diff
@@ -28,11 +32,15 @@ public void shouldNotRecordSuccessTestExecutionByDefault() {
 
         //Clean the field
         outputFile.delete();
+
+        restoreRecorderOption(oldMode);
```

Comment:
Yes I thought the same, but the problem is that I think that we might need to set different options in each test, not done yet because I want to see if this works when is executed in Jenkins, and if it works I will need to set different values in some tests. So although now seems that it is the same code, it will change in near future.